src/config/settings.py

```
@@ -1,134 +1,58 @@
-    """
-    Create and cache settings instance.
-    Returns:
-        Settings: Application settings instance
-    """
-    try:
-        return Settings()
-    except ValidationError as e:
-        logging.error(f"Configuration error: {e}")
-        raise
+# Global configuration instance
+config = Config()
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)